lib/timescaledb/models/model.ex

```diff
@@ -92,4 +92,8 @@ defmodule Membrane.Telemetry.TimescaleDB.Model do
   def add_link(link) do
     Link.changeset(%Link{}, link) |> Repo.insert()
   end
+
```

Comment:
There are private functions before public ones in this file what looks strange. Additionally, it lacks specs and docs.

Reply:
I don't think that specs and docs in this case are really necessary. Those should be a part of a public interface that some user can use and in this case everything happens internally and everything is self contained.

I can understand moduledoc for the models to explain why they are designed as they are but providing docs and specs for inserting a model to the table are not much of use.